### ISSUE TITLE
Extremely long startuptime after reboot

### ISSUE THREAD

**Issue by xelra** [equalsraf/neovim-qt#250]
This might be specific to Windows.

After rebooting, the first startup of Neovim-Qt takes an extremely long time. I've measure it to be 11 seconds on my system.

Startups after that take less than a second.

**Comment by justinmk** [comment]
Do you have a network drive in $PATH?

**Comment by xelra** [comment]
No, no network drive in `%PATH%`.

**Comment by justinmk** [comment]
Any network drives at all? If you disconnect them, does the problem persist?

**Comment by xelra** [comment]
I do not have a single network drive mapped.

**Comment by justinmk** [comment]
@xelra Does nvim or nvim-qt live in a path that contains spaces? Does the problem occur if you put it somewhere without spaces? E.g. `C:/Users/foo/nvim`.

I noticed some slowness when the path contained spaces. No idea why.

**Comment by xelra** [comment]
Yes, it's connected to being installed to a path with spaces. I moved the whole Neovim-Qt folder from `C:\Program Files (x86)\Neovim-Qt` to `C:\Users\xelra\Desktop\Neovim-Qt` and then it starts up way faster.

So after a reboot and/or after waiting some time (I couldn't determine how much time yet; maybe 30 minutes or one hour) it has extremely slow startup time, when executed from a path with spaces.

**Comment by justinmk** [comment]
Thanks. Could you update the issue title?
